easybuild/easyblocks/b/blender.py

```diff
@@ -39,6 +39,13 @@
 class EB_Blender(CMakeMake):
     """Support for building Blender."""
 
+    @staticmethod
+    def extra_options():
+        extra_vars = CMakeMake.extra_options()
+        extra_vars['separate_build_dir'][0] = True
+        extra_vars['build_type'][0] = 'Release'
```

Comment:
If toolchainopts debug is True this should probably be Debug

Reply:
This is now done in the CMakeMake EB and this line has been removed